tests/test_rocket_math.py

```diff
@@ -0,0 +1,752 @@
+# Test suite for rocket_math.py
+
+import numpy as np
+
+import rocket_math as rm
+
+'''
+Notes:
+- For all calculations, there will be an arbitrary 4 decimal places max.
+- No rocket_air_density unit test since the current method for
+determining/testing air density needs to be changed.
+'''
+
+# Blank Rocket object to be used to initialize rockets in functions.
```

Comment:
big problem with this - you only create one Rocket object, and it's all passed by reference, so every function that you create "test_rocket" is still pointing to this blank_rocket. all modifications you make are global. assuming this isn't intended, you'll need to instantiate a new Rocket for each function